endpoints.json

```diff
@@ -23,5 +23,82 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id": {
+    "description": "serves the article with the requested ID",
+    "queries": [],
+    "exampleResponse": 
+      
+        {
+          "title": "Seafood substitutions are increasing",
+          "topic": "cooking",
+          "author": "weegembump",
+          "body": "Text from the article..",
+          "created_at": 1527695953341
+        }
+      
+    }
+  ,
+
+  "GET /api/articles/:article_id/comments":{
+    "description": "serves an array of all comments for requested ID", 
+    "queries": [],
+    "exampleResponse": [
+      
+      {
+        "body": "git push origin master",
+        "votes": 0,
+        "author": "icellusedkars",
+        "comment_id": 10,
+        "article_id": 3,
+        "created_at": "2020-06-20T07:24:00.000Z"
+      }
+      ]
+    }
+  ,
+
+  "POST /api/articles/:article_id/comments":{
+    "description": "serves an array of all comments for requested ID", 
+    "queries": [],
+    "exampleResponse": [
+      { "comment": "Great article!"}
+    ]
+  }
```

Comment:
I dont think this example response is exactly accurate? you will also need an exampleBody here which shows what is expect to be sent by the user for example an object with a valid user and body